src/test/java/com/klarna/hiverunner/HiveServerContainerTest.java

```diff
@@ -15,7 +15,9 @@
  */
 package com.klarna.hiverunner;
```

Comment:
This file hasn't really been changed? Just noise so perhaps remove from this PR?